src/pages/todoSetup/molecules/todoList/atoms/todoItem/TodoItem.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Lodash
+import _noop from 'lodash/noop';
+
+// Constants
+import { EMPTY_OBJECT } from '../../../../../../constants/todo.general'
+
+//Styles
+import "./TodoItem.css"
+
+const TodoItem = ({todoInfo, onTodoItemClick }) => {
+
+    const {id, text, isCompleted} = todoInfo;
+
+    const onTodoClick = () => {
+        onTodoItemClick(id)
+    }
+
+    const strikeClass = isCompleted ? "strike-class" : " ";  
+
+    return (
+        <div className='todo-item'>
+            <div className={`todo-value ${strikeClass}`} onClick={onTodoClick}>{text}</div>
```

Comment:
SRP : create separate function to get Class name : getTodoItemClass = ()=>{return className}